content/rippled-api-methods/account_lines.md

```diff
@@ -172,6 +172,8 @@ Each trust line object has some combination of the following fields:
 | `quality_out`    | Unsigned Integer | Rate at which the account values outgoing balances on this trust line, as a ratio of this value per 1 billion units. (For example, a value of 500 million represents a 0.5:1 ratio.) As a special case, 0 is treated as a 1:1 ratio. |
 | `no_ripple`      | Boolean          | (May be omitted) `true` if this account has enabled the [NoRipple flag](concept-noripple.html) for this line. If omitted, that is the same as `false`. |
 | `no_ripple_peer` | Boolean          | (May be omitted) `true` if the peer account has enabled the [NoRipple flag](concept-noripple.html). If omitted, that is the same as `false`. |
+| `authorized`     | Boolean          | (May be omitted) `true` if this account has set the tfSetAuth flag on this trust line. If omitted, that is the same as `false`. |
```

Comment:
We should consider linking to the appropriate docs on the authorized trust lines thing. Also, nitpicky, but technically the "tfSetAuth flag" is part of the TrustSet transaction, not the trust line; on the trust line, it's the lsfLowAuth or lsfHighAuth flag (depending on which side of the trust line is mathematically lower or higher).

Some candidates for sections to link... https://ripple.com/build/ledger-format/#ripplestate-flags
https://ripple.com/build/gateway-guide/#authorizing-trust-lines
https://ripple.com/build/gateway-guide/#authorized-accounts

(I'll circle back to this and edit it myself tonight or tomorrow if you don't mind.)

Reply:
Please do. Thanks!